filament/backend/src/vulkan/VulkanPipelineCache.h

```diff
@@ -184,6 +186,25 @@ class VulkanPipelineCache : public CommandBufferObserver {
 private:
     static constexpr uint32_t ALL_COMMAND_BUFFERS = (1 << VK_MAX_COMMAND_BUFFERS) - 1;
 
+    struct LayoutBundleKey {
+        // Currently, only samplers can be bound to each shader stages.
+        std::array<VkShaderStageFlags, SAMPLER_BINDING_COUNT> samplers;
+    };
+
+    static LayoutBundleKey getLayoutBundleKey(const Program::SamplerGroupInfo& samplerGroupInfo) noexcept;
+
+    struct LayoutBundle {
+        std::array<VkDescriptorSetLayout, DESCRIPTOR_TYPE_COUNT> setLayouts;
+        VkPipelineLayout pipelineLayout;
+        uint32_t reference = 0;
```

Comment:
I would call this `referenceCount`.

Reply:
Ok it's changed.

Reply:
It looks like it is still called `reference`.